.github/ISSUE_TEMPLATE/10_bug_report.yml

```diff
@@ -38,6 +43,18 @@ body:
     validations:
       required: true
 
+  - type: checkboxes
+    id: new-profile
+    attributes:
+      label: New profile
+      description: |
+        Did you try with a new QGIS Profile ? Some issues or crash might be related to plugins or user configuration.
```

Comment:
```suggestion
        Did you try with a new QGIS Profile? Some issues or crashes might be related to plugins or user configuration.
```

Reply:
Can we decapitalize "profile", there is no reason for it to have a capital P.